helpers/decorators.py

```diff
@@ -17,6 +17,8 @@ def inner(*args, **kwargs):
             return error.message
         except NameValueError as error:
             return error.message
+        except BirthdayValueError as error:
```

Comment:
Please add 
except DateValueError as error:
            return error.message
to input_errors